core/templates/dev/head/app.js

```diff
@@ -71,6 +71,15 @@ oppia.constant(
   'RTE_COMPONENT_SPECS',
   window.GLOBALS.RTE_COMPONENT_SPECS ? window.GLOBALS.RTE_COMPONENT_SPECS : {});
 
+oppia.constant(
+  'AUDIO_UPLOAD_URL_TEMPLATE',
+  '/createhandler/audioupload/<exploration_id>');
+
+oppia.constant(
+  'GCS_AUDIO_DOWNLOAD_URL_TEMPLATE',
+  'https://storage.googleapis.com/oppiatestserver-resources/' +
```

Comment:
I'm not sure I'm getting what you're saying. Is it okay if I just move this all into a variable inside AssetsBackendApiService?